crates/fsrt/src/test.rs

```diff
@@ -0,0 +1,71 @@
+use crate::{scan_directory_test, MockForgeProject};
+use forge_loader::manifest::{ForgeManifest, FunctionMod};
+use std::{collections::HashMap, path::PathBuf};
+
+#[test]
+fn test_simple() {
+    let forge_manifest = ForgeManifest::create_manifest_with_func_mod(FunctionMod {
+        key: "main",
+        handler: "index.run",
+        providers: None,
+    });
+
+    let mut test_forge_project = MockForgeProject {
+        test_manifest: forge_manifest,
+        files_name_to_source: HashMap::new(),
+    };
+    test_forge_project.files_name_to_source.insert(
```

Comment:
You should probably have a function called `fn add_file(&mut self, p: impl Into<PathBuf>, source: impl ToString)` to `MockForgeProject` where you add the path and the source to the inner soucemap and HashMap. That way, you don't have to change all the tests, whenever we change the implementation of `MockForgeProject`.

Reply:
Fixed. Thank you :)